packages/p2p/src/pages/app.jsx

```diff
@@ -250,7 +250,6 @@ const App = () => {
             order_store.setVerificationCode(code_param);
         }
         if (action_param && code_param) {
-            general_store.showModal({ key: 'LoadingModal', props: {} });
```

Comment:
so we don't need to show the loading modal here ?

Reply:
there's no need for this